pkg/api/summary.go

```diff
@@ -5,29 +5,24 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"time"
 
 	"github.com/wakatime/wakatime-cli/pkg/summary"
 )
 
-const dateFormat = "2006-01-02"
-
-// Summaries fetches summaries for the defined date range.
+// Summary fetches code stats for Today.
 //
 // ErrRequest is returned upon request failure with no received response from api.
 // ErrAuth is returned upon receiving a 401 Unauthorized api response.
 // Err is returned on any other api response related error.
-func (c *Client) Summaries(startDate, endDate time.Time) ([]summary.Summary, error) {
-	url := c.baseURL + "/users/current/summaries"
+func (c *Client) Summary() (*summary.Summary, error) {
```

Comment:
Maybe get rid of `summary`? 
```suggestion
func (c *Client) Today() (*summary.Summary, error) {
```